import_fixer.py

```diff
@@ -0,0 +1,81 @@
+import os
+import shutil
+from tempfile import mkstemp
+
+import_statement = 'import '
+
+def split_to_list(drive_and_path):
+    path = os.path.splitdrive(drive_and_path)[1]
+    folders = []
+    while 1:
+        path, folder = os.path.split(path)
+
+        if folder != "":
+            folders.append(folder)
+        else:
+            if path != "":
+                folders.append(path)
+
+            break
+
+    folders.reverse()
+    return folders
+
+
+def get_deepness(top_level_dir, path_list):
+    return len(path_list) - path_list.index(top_level_dir)
+
+
+def analyze_file(deepness, file_path, top_level_import):
+    replace_map = dict()
+    fh, abs_path = mkstemp()
+    modified = False
+    with os.fdopen(fh, 'w') as new_file:
+        with open(file_path, 'r') as old_file:
+            lines = old_file.readlines()
+            for i in range(len(lines)):
+                line = lines[i]
+                extra_cases = '.' in line and top_level_import in line and not (import_statement + '.') in line
+                if line.startswith(import_statement) and extra_cases:
+                    cut_line = line[len(import_statement):].rstrip()
+                    ending_string = cut_line[cut_line.rfind('.') + 1:]
+                    replace_map[cut_line] = ending_string
+                    line = 'from ' + '.' * deepness + cut_line[:cut_line.rfind(ending_string) - 1] + ' import ' + ending_string + '\n'
+                    modified = True
+                else:
+                    for key in replace_map:
+                        if key in line:
+                            modified = True
+                            line = line.replace(key, replace_map[key])
+                new_file.write(line)
+    if modified:
+        os.remove(file_path)
+        shutil.move(abs_path, file_path)
+        print(str(len(replace_map)), 'imports')
+    else:
+        print('not modified')
+
+
+def prevent_leaks(top_level_dir='generated', exclude_dir=None, top_level_import="api"):

```

Comment:
@dtracers Is there a danger of creating leaks if protobuffs are imported the wrong way? 

Not gonna try to understand this file, but just curious going by the file and function name if there are any new landmines unique to protobuf usage that need to be known.